ublox_gps/package.xml

```diff
@@ -26,6 +26,7 @@
   <depend>tf2</depend>
   <depend>ublox_msgs</depend>
   <depend>ublox_serialization</depend>
+  <depend>mavros_msgs</depend>
```

Comment:
Is this not sufficient for the package.xml dependency? Is there something else that needs to be added for the buildfarm?